server_admin/topics/roles.adoc

```diff
@@ -1,9 +1,4 @@
 == Roles
+A role is a category of user. An application can assign access and permissions to a role and then assign multiple users to that rule, so that the users all share the same access and permissions. For example, the Admin Console has roles that give permission to users to access parts of the Admin Console UI.
```

Comment:
Instead of "A role is a category of user."  How about  "A role typically applies to one type of user."